crates/tests/src/integration/ledger_tests.rs

```diff
@@ -1578,3 +1580,218 @@ fn change_validator_metadata() -> Result<()> {
 
     Ok(())
 }
+
+// Test that fee payment is enforced and aligned with process proposal. The test
+// generates a tx that subtract funds from the fee payer of a following tx. Test
+// that wrappers (and fee payments) are evaluated before the inner transactions.
+#[test]
+fn enforce_fee_payment() -> Result<()> {
```

Comment:
update `.github/workflows/scripts/e2e.json` to run this in CI

Reply:
ah sry, nvm it's integration

Reply:
Isn't that file for e2e tests only? I see from the CI logs that the test runs